pandas/tests/test_resample.py

```diff
@@ -252,6 +252,15 @@ def test_pipe(self):
         result = r.pipe(lambda x: x.max() - x.mean())
         tm.assert_frame_equal(result, expected)
 
+    def test_groupby_resample_on_api_with_getitem(self):
```

Comment:
can you move this a bit lower down in the file (immediately after the other getitem tests)